app/templates/play/campaign-view.jade

```diff
@@ -22,25 +22,21 @@ if showGameDevAlert
 
 // TODO: .gameplay-container causes world map buttons to briefly appear in top left of screen
 .gameplay-container
-  if features.codePlay
```

Comment:
Here it looks like a lot changed, but it's only un-indenting the block below as the entire thing is in the `else`